### ISSUE TITLE
Layout/IndentationWidth with assertion

### ISSUE THREAD

**Issue by kvokka** [rubocop/rubocop#9696]
# Expected behavior

That auto-corrector will convert this

```ruby
foo = if true
23
else
42
end
```

to this

```ruby
foo = if true
        23
      else
        42
      end
```

This problem appears only in combination assertion with if statement 

## Actual behavior

Instead i get this:

```ruby
foo = if true
  23
          else
            42
  end
```

## RuboCop version

```
1.12.1 (using Parser 3.0.1.0, rubocop-ast 1.4.1, running on ruby 2.7.2 x86_64-darwin19)
```

**Comment by koic** [comment]
This is not auto-corrected to the expected code with `Layout/IndentationWidth` only. Please apply `Layout/ElseAlignment` and `Layout/EndAlignment` together. Thank you.

## Before

```ruby
% cat example.rb
foo = if true
23
else
42
end
```

## Arrange

```console
% bundle exec rubocop -a
(snip)

Inspecting 1 file
W

Offenses:

example.rb:1:1: W: Lint/UselessAssignment: Useless assignment to variable - foo.
foo = if true
^^^
example.rb:1:1: C: [Correctable] Style/FrozenStringLiteralComment: Missing frozen string literal comment.
foo = if true
^
example.rb:1:10: W: Lint/LiteralAsCondition: Literal true appeared as a condition.
foo = if true
         ^^^^
example.rb:2:1: C: [Corrected] Layout/IndentationWidth: Use 2 (not -6) spaces for indentation.
23 ...
^^
example.rb:3:1: C: [Corrected] Layout/ElseAlignment: Align else with if.
else
^^^^
example.rb:4:1: C: [Corrected] Layout/IndentationWidth: Use 2 (not 0) spaces for indentation.
42

example.rb:4:3: C: [Corrected] Layout/IndentationWidth: Use 2 (not -4) spaces for indentation.
  42 ...
  ^^
example.rb:5:1: W: [Corrected] Layout/EndAlignment: end at 5, 0 is not aligned with if at 1, 6.
end
^^^

1 file inspected, 8 offenses detected, 5 offenses corrected, 1 more offense can be corrected with `rubocop -A`
```

## After

```ruby
% cat example.rb
foo = if true
        23
      else
        42
      end
```

Tip: The same result can be obtained with `rubocop -a --only Layout/IndentationWidth,Layout/ElseAlignment,Layout/EndAlignment`. Thank you.

**Comment by kvokka** [comment]
Thank you so much for such a quick and full response!
